.github/workflows/publishing.yml

```
@@ -844,8 +868,7 @@ jobs:
           source validate_wheel.sh -w /tmp/install/cuda_quantum-*.whl -f /tmp -p 3.10
           set -e # Re-enable exit code error checking
           if [ "$status_sum" -ne "0" ]; then
-            echo "Error running validation script"
-            exit $status_sum
+            echo "::error::Error running validation script" && exit $status_sum
```

```suggestion
            echo "::error::Error running validation script" ; exit $status_sum
```
Don't we want this to exit unconditionally at this point?

---

I am not sure what you mean? This exits unconditionally as well?

---

If `echo` fails with a write error, then the `$status_sum` may not be returned with the `exit`.

---

Fair point. I updated that across the board.